Classes/Event/EventBus.php

```diff
@@ -1,54 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\Cqrs\Event\Exception\EventBusException;
-use Neos\Cqrs\EventListener\EventListenerLocator;
-use TYPO3\Flow\Annotations as Flow;
-use TYPO3\Flow\Log\SystemLoggerInterface;
-use TYPO3\Flow\Utility\TypeHandling;
-
-/**
- * @Flow\Scope("singleton")
- */
-class EventBus
```

Comment:
> What does stripping the EventBus has to do with allowing metadata enrichment?

Nothing directly, but the changes rendered it redundant.

@robertlemke @albe can you think of a single scenario that requires the EventBus in its current form? We could rename `EventPublisher` to `EventBus` if that makes it better for you!?